src/index.js

```diff
@@ -0,0 +1,7 @@
+import Game from './game';
+
+window.addEventListener('load', () => {
+  const size = 4;
+  const game = new Game(size);
+  game.init(size);
```

Comment:
I don't see in class realisation that `init()` method receives any arguments

Reply:
fixed